### ISSUE TITLE
Date time does not work

### ISSUE THREAD

**Issue by bmentink** [helloSystem/hello#175]
I set my correct time zone during installation, but the Date/Time is incorrect on my installed OS. There does not seem to be any way to correct it, or set to NPT time ..

**Comment by kettle-7** [comment]
Yeah, I have the same problem. Can you please open a Terminal window (click on the desktop and push F4), and type in `tzsetup`, press enter, follow the instructions, and tell me what the timezone says? It'll say something like 'Pacific/Auckland'.

**Comment by kettle-7** [comment]
Sorry, you need `sudo tzsetup`, it's different on FreeBSD.

It only just occurred to me that I haven't used the FreeBSD tzsetup yet, but I did everything manually in the installation.

**Comment by bmentink** [comment]
Hi, I tried running "sudo tzsetup", it did not report the current time zone, rather it directed me to set it up, which I did.
I set it to NZST (New zealand standard time), but the time/date in the gui remains the same, I will try logging out and back in again to see if it has changes.

PS: Typing "date" in the terminal shows the same incorrect date ..

**Comment by bmentink** [comment]
I logged out and back in again, but still no change. It's like NPT is not running, how do I check?
